packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -7589,17 +7589,26 @@ export class AdapterClass extends EventEmitter {
     registerNotification<Scope extends keyof ioBroker.NotificationScopes>(
         scope: Scope,
         category: ioBroker.NotificationScopes[Scope] | null,
-        message: string
+        /** Message for messangers */
```

Comment:
typo in `messangers` but I would avoid being specific to messaging adapters etc as this is more generic

Reply:
Fixed